app/assets/stylesheets/billboards.scss

```diff
@@ -19,7 +19,16 @@
   }
 }
 
-.sponsorship-dropdown {
+.billboard .crayons-dropdown.sponsorship-dropdown {
```

Comment:
I had to increase the specificity to be ensure that it overrides the default values for the crayons-dropdown.